### PR TITLE
✨ Create crate search script

### PR DESCRIPTION
**Pull request by Matt-Gleich**
## Description

Added a new script command that will take a crate name and automatically search for it on crates.io.

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

https://user-images.githubusercontent.com/43759105/111051517-9dbb0a00-8421-11eb-83ef-25828049ef61.mov

## Dependencies / Requirements

Nothing is required

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/rust-crate.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# @raycast.schemaVersion 1
+# @raycast.title Search for Crate
+# @raycast.mode silent
+# @raycast.packageName Rust
+# @raycast.argument1 { "type": "text", "placeholder": "name", "percentEncoded": true }
+#
+# Optional parameters:
+# @raycast.icon ./images/cargo-logo.png
+# @raycast.currentDirectoryPath ~
+# @raycast.needsConfirmation false
```
Comment (unnamedd) [inline commands/developer-utils/rust-crate.sh:13]: @Matt-Gleich both `currentDirectoryPath` and `needsConfirmation` isn’t needed here.

File: commands/developer-utils/rust-crate.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (unnamedd) [inline commands/developer-utils/rust-crate.sh:1]: This Script Command must be in `web-searches`.

File: commands/developer-utils/rust-crate.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search for Crate
+# @raycast.mode silent
+# @raycast.packageName Rust
```
Comment (unnamedd) [inline commands/developer-utils/rust-crate.sh:7]:
```suggestion
# @raycast.packageName Web Seaches
```

**Review by unnamedd — CHANGES_REQUESTED** [review]

**Comment by Matt-Gleich** [comment]
Thanks for the review! Just made those changes.

**Review by unnamedd — APPROVED** [review]
Great @Matt-Gleich! Looks good to me!

File: commands/web-searches/rust-crate.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search for Crate
+# @raycast.mode silent
+# @raycast.packageName Web Search
+# @raycast.argument1 { "type": "text", "placeholder": "name", "percentEncoded": true }
```
Comment (thomaspaulmann) [inline commands/web-searches/rust-crate.sh:8]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Name", "percentEncoded": true }
```

**Pull request merged**